README.md

```diff
@@ -1,21 +1,25 @@
-This repository contains the first alpha Nextflow pipeline allowing to split contigs according to their predicted kingdom. 
+# kingdom-investigator
 
-## Requirements
+This repository contains a Nextflow pipeline allowing to isolate eukaryotic contigs from assemblies. 
 
-The current prerequisites are Singularity and the `deepmicroclass.sif` & `tiara.sif` associated tool images, which are located in :
+## Requirements
 
-`/hps/nobackup/rdf/metagenomics/service-team/users/louison/nextflow/project/bin/`
+The current prerequisites are Singularity and the tool images of [Tiara](https://github.com/ibe-uw/tiara), [DeepMicroClass](https://github.com/chengsly/DeepMicroClass/tree/master) and [CAT](https://github.com/MGXlab/CAT_pack). 
 
 ## Usage
 
-You can run the current version of the pipeline like this:
+You can run the current version of the pipeline like this :
+
+    nextflow run main.nf -profile ebi_slurm --contigsPath {pathToFasta}
+
+For non-EBI users, note that you'll need to create your own profile and associated config file in order to precise the container paths and versions. 
 
-    nextflow run main.nf --contigsFile {fastaContainedInData.fa}
+## Settings
 
 Note that you can specify the [DeepMicroClass](https://github.com/chengsly/DeepMicroClass/tree/master) parameters by adding these when running the nextflow script :
 
-    nextflow run main.nf --contigsFile {fastaContainedInData.fa} --model {path} --encoding {onehot,embedding} --mode {hybrid,single} --singleLen {n} --device {cpu,cuda}
+    nextflow run main.nf -profile ebi_slurm --contigsPath {pathToFasta} --model {path} --encoding {onehot,embedding} --mode {hybrid,single} --singleLen {n} --device {cpu,cuda}
 
 By default, the model contained in the Singularity image `deepmicroclass.sif` is used.
 
-In addition, although it is possible to specify the use of CUDA, please first check that the image used corresponds to the GPU version of DeepMicroClass. 
\ No newline at end of file
+In addition, although it is possible to specify the use of CUDA, please first check that the used image corresponds to the GPU version of DeepMicroClass.
```

Comment:
An image of your pipeline + decisions it's making if you have one would be useful? - save it in the repo in a figures directory as well.

Reply:
Excellent idea ! I'll probably add a Mermaid diagram (DAG) and a schema summarizing the decision when we will have confirmed the precise stages of the pipeline :)